books/u-physics/bake

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '2.0.0'
```

Comment:
update this to a more recent release

Reply:
sure! there isn't a release yet that has _everything_ in it but i'll put the most recent